### PR TITLE
blog: fix grammar, spelling and whitespace in Markdown files

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: blog/2024-01-08-cloudstack-and-gitops-at-scale/index.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ user/contributor community like what we found with CloudStack.
   other technologies. What do you recommended to be tested during a
   PoC?</strong>
 
-Why you chose CloudStack?   Each use case is different obviously. The
+Why you chose CloudStack?  Each use case is different obviously. The
```
Comment (jbampton) [inline blog/2024-01-08-cloudstack-and-gitops-at-scale/index.md:78]:
```suggestion
Why you chose CloudStack? Each use case is different obviously. The
```

File: blog/2022-11-30-cloudstack-collaboration-conference-2022-roundup.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ In this talk, Vladi looks at the new Volume encryption option (due in CloudStack
 <div></div>
 
 &nbsp;
-<div><span ><strong><a title="StorPool Storage Оverview and Integration with CloudStack" href="https://www.slideshare.net/ShapeBlue/storpool-storage-verview-and-integration-with-cloudstack" target="_blank" rel="noopener">StorPool Storage Оverview and Integration with CloudStack</a> - Slavka Peleva, Venko Moyankov, Alex Ivanov </strong></span></div>
+<div><span ><strong><a title="StorPool Storage Оverview and Integration with CloudStack" href="https://www.slideshare.net/ShapeBlue/storpool-storage-verview-and-integration-with-cloudstack" target="_blank" rel="noopener">StorPool Storage Оverview and Integration with CloudStack</a> - Slavka Peleva, Venko Moyankov, Alex Ivanov</strong></span></div>
```
Comment (DaanHoogland) [inline blog/2022-11-30-cloudstack-collaboration-conference-2022-roundup.md:286]:
```suggestion
<div><span><strong><a title="StorPool Storage Оverview and Integration with CloudStack" href="https://www.slideshare.net/ShapeBlue/storpool-storage-verview-and-integration-with-cloudstack" target="_blank" rel="noopener">StorPool Storage Оverview and Integration with CloudStack</a> - Slavka Peleva, Venko Moyankov, Alex Ivanov</strong></span></div>
```

**Review by DaanHoogland — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@jbampton is there any pre-commit check to come with this?

**Comment by jbampton** [comment]
Hi @DaanHoogland no pre-commit checks with this PR but I can add more in following PRs

**Pull request merged**